code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -995,9 +1007,30 @@
 
 /obj/item/weapon/gun/energy/yautja/plasmapistol/delete_bullet(obj/item/projectile/projectile_to_fire, refund = 0)
 	qdel(projectile_to_fire)
-	if(refund) charge_time *= 2
+	if(refund)
+		charge_time *= 2
```

Comment:
This is not how this should work if it is refunding a single bullet, should add the shot cost